mcos/error_estimator/base.py

```diff
@@ -2,13 +2,16 @@
 import numpy as np
 
 
-class ErrorEstimator(ABC):
+class AbstractErrorEstimator(ABC):
     @abstractmethod
-    def estimate(self, allocation: np.array) -> float:
+    def estimate(self, mu: np.array, cov: np.array, allocation: np.array, optimal_allocation: np.array) -> float:
```

Comment:
It looks like some of the other estimation methods require the covariance matrix, so I've added it as a parameter.